Fix lastpass lookup plugin on python 3

dannyim opened the pull request:
- Open files in text mode instead of binary mode

##### SUMMARY
<!--- Describe the change, including rationale and design decisions -->
Fixes #42062, fixing the lastpass lookup plugin for python 3
<!--- If you are fixing an existing issue, please include "Fixes #nnn" in your
commit message and your description; but you should still explain what
the change does.-->

##### ISSUE TYPE
<!--- Pick one below and delete the rest: -->
 - Bugfix Pull Request

##### COMPONENT NAME
<!--- Name of the module, plugin, module or task -->
lastpass
##### ANSIBLE VERSION
<!--- Paste verbatim output from "ansible --version" between quotes below -->
```
ansible 2.6.0 (issue_42062_bugfix e6936f02e8) last updated 2018/06/28 13:04:38 (GMT -500)
  config file = None
  configured module search path = ['/home/imxxx021/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /c/Users/imxxx021/wsl/projects/ansible/lib/ansible
  executable location = /c/Users/imxxx021/wsl/projects/ansible/bin/ansible
  python version = 3.5.2 (default, Nov 23 2017, 16:37:01) [GCC 5.4.0 20160609]
```


##### ADDITIONAL INFORMATION
<!--- Include additional information to help people understand the change here.
For bugs that don't have a linked bug report, a step-by-step reproduction
of the problem is helpful. -->

<!--- Paste verbatim command output below, e.g. before and after your change -->
```

```

Commit:
Fix lastpass lookup plugin on python 3

- Use to_text and to_bytes

hedinfaok commented:
I ran into the same bug today, and your fix works as expected. Thanks @dannyim !

abadger reviewed (commented):
This is not the fix that I would suggest (universal newlines can have other side effects).  Instead, use ansible.module_utils._text.to_text to convert the data returned from bytes to text.

dannyim commented:
Thanks for the suggestion, I updated the fix to use the text functions in module_utils. I also updated the mock within the test code to return byte strings instead of plain text strings.

marcomc commented:
anyone would like to review this PR soon hopefully? many many thanks

Akasurde commented:
superseded by https://github.com/ansible/ansible/pull/48804
